app/controllers/UserDataController.scala

```diff
@@ -0,0 +1,46 @@
+package controllers
+
+import com.gu.facia.client.models.Trail
+import com.gu.scanamo._
+import com.gu.scanamo.syntax._
+import services.Dynamo
+import model.UserData
+
+class UserDataController(dynamo: Dynamo, val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) {
+  import model.UserData._
+
+  private lazy val userDataTable = Table[UserData](config.faciatool.userDataTable)
```

Comment:
Not for now, but would it be worth thinning out our controllers by creating a UserDataDAO/UserDataService or equivalent and delegating persistence duties to that, especially as our persistence requirements for user data grow?

Reply:
Yep, I thought of this as well and I thought when we get a third type of user data it will be worth doing it.